scripts/combine_lightcurves.py

```diff
@@ -1,43 +1,93 @@
+#!/usr/bin/env python3
+
 import os
+import shutil
 import json
 import argparse
+from json.decoder import JSONDecodeError
 
-import glob
+from pocs.utils.google.storage import PanStorage
 
 
 def get_curves_for_pic(pic):
+    """Read light curve files and add all curves to array.
+
+    :param pic: the PIC id for the star to make a master light curve for
+    :return: an array of all the light curves for the given PIC
+    """
     curves = []
-    for filename in glob.iglob("data/LC/{}/**/*.json".format(pic), recursive=True):
-        with open(filename, 'r') as f:
-            c = json.load(f)
-            curves.append(c)
-    return curves
+    bucket_name = 'panoptes-simulated-data'
+    prefix = "LC/{}/".format(pic)
+    temp_dir = '{}/temp'.format(os.getenv('PANDIR', default='/var/panoptes'))
```

Comment:
agree that i should be using /tmp and a less collidable dir name. 

i removed the cleanup functionality in #16, because currently this script is run every time a new light curve is added to the bucket. and often a batch of data is uploaded together, and removing the same temp dir while another process is trying to use it breaks things. i assume there's a better way to handle this, also because the same light curve is often being updated multiple times unnecessarily... not sure if that would be changing the design of when this script runs, or changing how temp data is stored